.circleci/config.yml

```diff
@@ -75,6 +75,10 @@ test: &test
         command: |
           bash <(curl -s https://codecov.io/bash) \
             -f /tmp/ibis/test-reports/pytest-cov/coverage.xml
+    - run:
```

Comment:
Please remove this, because We run multiple backends on the CI, and it can be confusing to only highlight the logs of mapd on failure.